pkg/targets/reconciler/tektontarget/reaper.go

```diff
@@ -71,7 +71,7 @@ func reaperThread(ctx context.Context, r *reconciler) {
 					continue
 				}
 
-				log.Info("Found target: " + t.Namespace + "." + t.Name)
+				log.Infow("Found target: " + t.Namespace + "." + t.Name)
```

Comment:
Do we really need `Infow` here as there isn't a `zap.Error(err)` being passed in as an arg?